deploy/base/clusterserviceversion.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
     operatorframework.io/suggested-namespace: security-profiles-operator
     operators.openshift.io/valid-subscription: '["OpenShift Kubernetes Engine", "OpenShift Container Platform", "OpenShift Platform Plus"]'
     operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.ppc64le: supported
+    operatorframework.io/os.linux: supported
```

Comment:
```suggestion
```

I believe this team support arm, so we can add leaves this out. If it doens't exist, it means all arches.